EST.MIT.InvoiceImporter.Function/Functions/ImporterFunctions.cs

```diff
@@ -12,7 +12,6 @@ namespace EST.MIT.InvoiceImporter.Function.Functions;
 public class ImporterFunctions : IImporterFunctions
 {
     private readonly IAzureBlobService _blobService;
-    private readonly BlobServiceClient _blobServiceClient;
     private readonly IAzureTableService _azureTableService;
 
     private readonly IEventQueueService _eventQueueService;
```

Comment:
I don;t think eventQueueService or notificationService are used in here any more. They can be removed as locals and in the constructor to tidy things up

Reply:
👍